static/scss/enterprise.scss

```diff
@@ -0,0 +1,193 @@
+// sass-lint:disable mixins-before-declarations
+
+.enterprise-page {
+  font-family: Rajdhani;
+}
+
+.nav-line {
+  background: var(--ColorLine, linear-gradient(270deg, #f0ba46 0%, #95bb83 51.02%, #3dbdbf 100%));
```

Comment:
We should either use the `var()` function properly i.e. to create variables and use those variables. At this point, we are just creating the variables and not using them. More details can be found in this [link](https://www.w3schools.com/css/css3_variables.asp#:~:text=var()%20Function-,The%20var()%20function%20is%20used%20to,value%20of%20a%20CSS%20variable.&text=Note%3A%20The%20variable%20name%20must,and%20it%20is%20case%20sensitive!).